I didn't manage to get the tests to pass.. something wrong with tf2_ros that need tf2_msgs.srv and didn't get it..

here is the output of the tests... 

## ERROR: Failure: ImportError (No module named srv)

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 411, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/lgeorge/fastSSD/ros/src/geometry_experimental/tf2_sensor_msgs/test/test_tf2_sensor_msgs.py", line 5, in <module>
    import tf2_sensor_msgs
  File "/home/lgeorge/fastSSD/ros/devel/lib/python2.7/dist-packages/tf2_sensor_msgs/**init**.py", line 35, in <module>
    exec(**fh.read())
  File "<string>", line 1, in <module>
  File "/home/lgeorge/fastSSD/ros/src/geometry_experimental/tf2_sensor_msgs/src/tf2_sensor_msgs/tf2_sensor_msgs.py", line 32, in <module>
    import tf2_ros
  File "/home/lgeorge/fastSSD/ros/devel/lib/python2.7/dist-packages/tf2_ros/__init**.py", line 35, in <module>
    exec(__fh.read())
  File "<string>", line 39, in <module>
  File "/home/lgeorge/fastSSD/ros/src/geometry_experimental/tf2_ros/src/tf2_ros/buffer.py", line 34, in <module>
    from tf2_msgs.srv import FrameGraph, FrameGraphResponse
ImportError: No module named srv

Can you help me with that ?

--- a/tf2_sensor_msgs/CMakeLists.txt
+++ b/tf2_sensor_msgs/CMakeLists.txt
@@ -33,9 +33,9 @@ add_executable(test_tf2_sensor_msgs_cpp EXCLUDE_FROM_ALL test/test_tf2_sensor_ms
 target_link_libraries(test_tf2_sensor_msgs_cpp ${catkin_LIBRARIES} ${GTEST_LIBRARIES})
 
 
-#if(TARGET tests)
-#  add_dependencies(tests test_tf2_sensor_msgs_cpp)
-#endif()
+if(TARGET tests)
+  add_dependencies(tests test_tf2_sensor_msgs_cpp)
+endif()
 add_rostest(${CMAKE_CURRENT_SOURCE_DIR}/test/test.launch)
 
 

